TestCases/parallel_regression.py

```diff
@@ -673,8 +673,8 @@ def main():
     turbmod_sa_neg_rae2822           = TestCase('turbmod_sa_neg_rae2822')
     turbmod_sa_neg_rae2822.cfg_dir   = "turbulence_models/sa/rae2822"
     turbmod_sa_neg_rae2822.cfg_file  = "turb_SA_NEG_RAE2822.cfg"
-    turbmod_sa_neg_rae2822.test_iter = 20
-    turbmod_sa_neg_rae2822.test_vals = [-2.004689, 0.742306, 0.497308, -5.265793, 0.809463, 0.062016]
+    turbmod_sa_neg_rae2822.test_iter = 10
```

Comment:
I am curious why a reduced number of iterations would be beneficial? I would always tend to do more as small errors might accumulate enough only after a certain amount of iterations, to break the regression test. That scenario might not happen to frequently but as you also remove multigrid there should be some time freed-up, right :)

So feel free (from my point of view) to revert to 20 or higher if it doesnt cost minutes.